cg_hermes/config/balsamic.py

```diff
@@ -1,601 +1,738 @@
+    "manta.all.vcf.gz.tbi": ["manta", "annotated-somatic-vcf-all-index", "sv", "vcf-all"],
+    "manta.all.vcf.gz_summary.html": [
+        "manta",
+        "sv",
+        "annotated-somatic-vcf-summary",
+        "vcf-summary",
+    ],
+    "manta.all.filtered.pass.vcf.gz": ["clinical-vcf-sv-pass", "vcf-sv-pass"],
+    "manta.all.filtered.pass.vcf.gz.tbi": ["clinical-vcf-sv-pass-index", "vcf-sv-pass"],
+    "tumor_normal.manta_germline.vcf.gz": [
+        "annotated-germline-vcf-all",
+        "sv",
+        "manta-germline",
+        "vcf-all",
+    ],
+    "tumor_normal.manta_germline.vcf.gz.tbi": [
+        "sv",
+        "annotated-germline-vcf-all-index",
+        "manta-germline",
+        "vcf-all",
+    ],
+    "tumor_normal.manta_germline.vcf.gz_summary.html": [
+        "annotated-germline-vcf-summary",
+        "sv",
+        "manta-germline",
+        "vcf-summary",
+    ],
+    # delly
+    "delly.all.vcf.gz": ["sv", "vcf-all", "delly", "annotated-somatic-vcf-all"],
+    "delly.all.vcf.gz.tbi": ["annotated-somatic-vcf-all-index", "sv", "vcf-all", "delly"],
+    "delly.all.vcf.gz_summary.html": [
+        "sv",
+        "annotated-somatic-vcf-summary",
+        "delly",
+        "vcf-summary",
+    ],
+    "delly.all.filtered.pass.vcf.gz": ["vcf-sv-pass-delly", "research-vcf-sv-pass-delly"],
+    "delly.all.filtered.pass.vcf.gz.tbi": ["vcf-sv-pass-delly", "research-vcf-sv-pass-delly-index"],
+    # cnvkit (PANEL)
+    "cnvkit.all.vcf.gz": ["cnv", "annotated-somatic-vcf-all", "cnvkit", "vcf-all"],
+    "cnvkit.all.vcf.gz.tbi": ["cnv", "cnvkit", "vcf-all", "annotated-somatic-vcf-all-index"],
+    "cnvkit.all.vcf.gz_summary.html": [
+        "cnv",
+        "cnvkit",
+        "annotated-somatic-vcf-summary",
+        "vcf-summary",
+    ],
+    "cnvkit.all.filtered.pass.vcf.gz": ["cnv", "research-vcf-cnv-pass", "vcf-cnv-pass"],
+    "cnvkit.all.filtered.pass.vcf.gz.tbi": ["cnv", "research-vcf-cnv-pass-index", "vcf-cnv-pass"],
+    "tumor.merged.cnr": ["cnv-cnr", "cnr"],
+    "tumor.merged.cns": ["cns", "cnv-cns"],
+    "tumor.merged-scatter.pdf": ["scatter", "cnv-scatter"],
+    "tumor.merged-diagram.pdf": ["cnv-diagram", "diagram"],
+    ".gene_metrics": ["gene-metrics", "cnv-gene-metrics"],
+    ".gene_breaks": ["cnv-gene-breaks", "gene-breaks"],
+    # TNscope_umi (PANEL)
+    "TNscope_umi.all.vcf.gz": ["tnscope-umi", "annotated-somatic-vcf-all", "snv", "vcf-all"],
+    "TNscope_umi.all.vcf.gz.tbi": [
+        "tnscope-umi",
+        "snv",
+        "vcf-all",
+        "annotated-somatic-vcf-all-index",
+    ],
+    "TNscope_umi.all.vcf.gz_summary.html": [
+        "tnscope-umi",
+        "snv",
+        "annotated-somatic-vcf-summary",
+        "vcf-summary",
+    ],
+    "TNscope_umi.balsamic_stat": ["tmb", "tnscope-umi", "snv", "stat-somatic-tmb"],
+    "TNscope_umi.all.filtered.vcf.gz": [
+        "research-vcf-filtered-tnscope-umi",
+        "vcf-filtered-tnscope-umi",
+        "snv",
+    ],
```

Comment:
```suggestion
    "TNscope_umi.all.filtered.vcf.gz": [
        "research-vcf-filtered-tnscope-umi",
        "vcf-filtered-tnscope-umi",
        "snv",
    ],
```
Isnt it good to have 4 fields in the tag list in order to have unique structure. Now why only 3 fields for this tag?

Reply:
I'm not sure about this because `"research-vcf-filtered-tnscope-umi"` can be interpreted as having 5 different tags. In any case, these tags are the BALSAMIC ones. If you consider it appropriate to add any additional, I could do it there (the tags that are linked to this particular file, for example, are defined inside the `bcftools_filter_TNscope_umi_tumor_normal` rule).